### ISSUE TITLE
Simplify how we translate strings outside components

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#2812]
For user-facing text whose source code is in the code of a component, `Label` and friends give us a nice convenient way to pass the text through translation.

When the code is in a function elsewhere, we have this way of doing it:
```
    const getString = value => this.context.intl.formatMessage({ id: value });
```
and then pass `getString` down the stack. We only do this in `MessageList` and its helpers, though -- everywhere else, we just don't translate. #1538 and #1539 cover two such areas.

This pattern is the right basic architecture, but it's wordy. The state we ought to get to is that basically every string in the source code that looks like it's meant for the user is clearly marked right there for translation. To do that, we'll be writing this a lot; we need to make it concise and low-noise.

This only needs two small changes:
* Change the name to `_`. This is a traditional name that's been widely used for this since at least the '90s, and the backend uses it.
* Put the handy wrapper (`getString` above, now to be called `_`) directly on `context`.

A good followup will be the equivalent of #1538 and #1539 with this style, and then to sweep to find untranslated strings that remain.

**Comment by gnprice** [comment]
See also https://github.com/zulip/zulip-mobile/issues/1481#issuecomment-348678750 and subsequent comments for some previous discussion.

**Comment by borisyankov** [comment]
I agree with the use of `context` for this vs what was used in the PRs. That is the reason I didn't merge earlier.

We need to figure out a more systemic approach to get everything translated. Would @kunall17 want to work on all the other related translation tasks?

**Comment by gnprice** [comment]
(See #2824 for a PR to do this.)
